tests/test-plugin-with-errors/test-plugin-with-errors.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Plugin Name: Test Plugin with Errors for Plugin Check
+ * Plugin URI: https://github.com/wordpress/plugin-check
+ * Description: Plugin Check plugin from the WordPress Performance Team, a collection of tests to help improve plugin performance.
+ * Requires at least: 6.0
+ * Requires PHP: 5.6
+ * Version: n.e.x.t
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ * Text Domain: test-plugin-check-errors
+ *
+ * @package test-plugin-check-errors
+ */
+
+include_once 'i18-usage-error-file.php';
```

Comment:
Can we use a single plugin file instead of multiple files? Also, can we test valid and invalid text domains in a single plugin instead of two plugins?

**Valid text domain:**
```
esc_html_e( 'Hello World!', 'test-plugin-check' );

$text_domain = 'test-plugin-check';
esc_html_e( 'Hello World!', $text_domain );
```

**Invalid text domain:**
```
esc_html_e( 'Hello World!', 'test-plugin-check-error' );

$text_domain = 'test-plugin-check-errors';
esc_html_e( 'Hello World!', $text_domain );
```

Reply:
@mukeshpanchal27 Actually, I have added two individual plugins as part of the infrastructure so that all other future checks' template test files can be added to their respective directories, like the "enqueued scripts sizes check" files, and so on.

Let me know if you need more information here. Thank you! 🙂